waspc/src/Wasp/Project/Analyze.hs

```diff
@@ -81,11 +77,11 @@ analyzeWaspFileContent prismaSchemaAst = return . left (map getErrorMessageAndCt
 constructAppSpec ::
   Path' Abs (Dir WaspProjectDir) ->
   CompileOptions ->
-  PackageJson ->
+  EC.ExternalConfigs ->
   Psl.Schema.Schema ->
   [AS.Decl] ->
   IO (Either [CompileError] AS.AppSpec, [CompileWarning])
-constructAppSpec waspDir options packageJson parsedPrismaSchema decls = do
+constructAppSpec waspDir options externalConfigs parsedPrismaSchema decls = do
```

Comment:
Ok, I get now why you introduced ExternalConfigs, to make it easier to forward tihs stuff. I guess that makes sense. Maybe it should go under Wasp.Project though hm.